xc/xc7/yosys/synth.tcl

```diff
@@ -30,6 +30,10 @@ if { $::env(USE_ROI) == "TRUE" } {
     # Start flow after library reading
     synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp -iopad -nowidelut -run prepare:check
 }
+
+# Check that post-synthesis cells match libraries.
+hierarchy -check -auto-top
```

Comment:
I think there shouldn't be `-auto-top` as this option may cause changing the top module.

Reply:
Sure, fixed